lib/matplotlib/axes/_axes.py

```diff
@@ -234,6 +256,10 @@ def set_ylabel(self, ylabel, fontdict=None, labelpad=None, **kwargs):
             Spacing in points from the axes bounding box including ticks
             and tick labels.
 
+        loc : {'left', 'center', 'right'}, default: :rc:`yaxis.titlelocation`
+            The label position. This is a high-level alternative for passing
```

Comment:
Pretty strongly feel this should be bottom/centrer/top.   This just seems like a typo.